docs/api.md

```diff
@@ -2487,6 +2488,45 @@ await page.mouse.move(0, 0);
 await page.mouse.up();
 ```
 
+#### Selecting text and moving the mouse
```

Comment:
I haven’t checked, but it could be that pptr.dev doesn’t deal with this header nicely (it might expect that each header corresponds with a method). In that case we could simply remove this header, making the content part of the `Mouse` class description.

Reply:
Good question and I wouldn't know. Let's see if it breaks when we publish and if yes, your solution sounds 👍

Reply:
Looks like it indeed shows up as a namespace: https://pptr.dev/#?product=Puppeteer&version=master&show=api-class-mouse We could update pptr.dev’s handling of the Markdown but since we’re reworking the docs anyway let’s just remove the header for now: 4205ff79d93c1d27b9cadde6f56c19be04397abd